tests/test_enum.rs

```diff
@@ -221,6 +221,44 @@ fn test_renaming_all_enum_variants() {
     });
 }
 
+#[pyclass(eq_int)]
+#[derive(Debug, Clone)]
+pub enum EqIntOnly {
+    VariantA,
+    VariantB,
+}
+
+#[test]
+fn test_simple_enum_eq_int_only() {
+    Python::with_gil(|py| {
+        let var1 = Py::new(py, EqIntOnly::VariantA).unwrap();
+        let var2 = Py::new(py, EqIntOnly::VariantA).unwrap();
+        let var3 = Py::new(py, EqIntOnly::VariantB).unwrap();
+        py_assert!(py, var1 var2, "var1 == var2");
+        py_assert!(py, var1 var3, "var1 != var3");
```

Comment:
Does this imply that by having `eq_int` that normal equality "works" anyway (but via integer conversion presumably)?

If so, maybe it makes sense to have `eq_int` require `eq` to make it not a surprise that value equality works? We can always relax that requirement later I guess.

Reply:
> Does this imply that by having `eq_int` that normal equality "works" anyway (but via integer conversion presumably)?

Yes, this is correct. 

> If so, maybe it makes sense to have `eq_int` require `eq` to make it not a surprise that value equality works? We can always relax that requirement later I guess.

Sure, I would love to have value comparisons only work with `eq`, but I did not see a good way to archive that.